test/runner/requirements/units.txt

```diff
@@ -9,3 +9,5 @@ python-memcached
 pyyaml
 redis
 unittest2 ; python_version < '2.7'
+jxmlease
+junos-eznc
```

Comment:
Lets chat more about this... im not sure this is the best route forward.  Can we add to this weeks IRC meeting agenda please?